doc/nrf/libraries/modem/gcf_sms_lib.rst

```diff
@@ -13,6 +13,25 @@ The library implements the following AT commands:
 * ``AT+CPMS``
 * ``AT+CSMS``
 * ``AT+CSCA``
+
+  * Set command: +CSCA=<sca>[,<tosca>]
```

Comment:
I do not know what would be a better place to describe `AT+CSCA` command.
It has to be described because AT commands from GCF SMS library are not in AT Command Reference Guide.